dessia_common/workflow/core.py

```diff
@@ -2186,6 +2186,44 @@ def _method_jsonschemas(self):
         jsonschemas['run_again']['classes'] = ["dessia_common.workflow.WorkflowRun"]
         return jsonschemas
 
+    def to_script(self):
+        """
+        Computes a script representing the workflowrun.
+        """
+        # TODO : don't take into account yet TypedVariable.
+        # TODO : take into account the type of inputs (type_)
+        workflow_script = self.workflow.to_script()
+        input_str = ""
+        default_value = ""
+        for j, block in enumerate(self.workflow.blocks):
```

Comment:
WorkflowRun knows its input_values. Couldn't we use this, instead of looping on both blocks and inputs ?